src/Illuminate/Console/Scheduling/CallbackEvent.php

```diff
@@ -43,6 +44,7 @@ public function __construct(EventMutex $mutex, $callback, array $parameters = []
         $this->mutex = $mutex;
         $this->callback = $callback;
         $this->parameters = $parameters;
+        $this->timezone = $timezone;
```

Comment:
There is no timezone property in the callback event class?

Reply:
It extends from the Illuminate\Console\Scheduling\Event class which has a timezone property

Reply:
Perfect.